.eslintrc

```diff
@@ -50,5 +50,13 @@
     "no-continue": "off",
     "no-await-in-loop": "off"
   },
-  "ignorePatterns": ["lib/*"]
+  "overrides": [
+    {
+      "files": ["test/*test.ts"],
+      "rules": {
+        "@typescript-eslint/no-non-null-assertion": "off"
+      }
+    }
+  ],
+  "ignorePatterns": ["lib/*", "src/*", "!src/index.ts"]
```

Comment:
Would it be possible to ignore from `src` only `mg-nft.ts` and `mg-market.ts`? This will allow us to lint any additional file we add to `src` in the future.

Reply:
done